_rules/aria-required-context-role-ff89c9.md

```diff
@@ -0,0 +1,257 @@
+---
+id: ff89c9
+name: ARIA required context role
+rule_type: atomic
+description: | 
+  This rule checks that a role exists inside its required context.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects: 
+- DOM Tree
+- CSS Styling
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Applicability
+
+The rule applies to any HTML or SVG element that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an explicit [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope), except if the element has an implicit semantic role that is identical to its explicit semantic role.
```

Comment:
I think an example would make this more readable:

>  has an explicit [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope) (i.e. `listitem` has `list` as its required context),

Reply:
Will try it out, but go for `tab` that has `tablist` as a required context, since most other roles (including `listitem`) has several possible context roles, which confuses the example.

Reply:
Example added as a note to match the other note that was already there to give an example on what it means that the implicit/explicit roles are different.